pkg/xds/envoy/tags/serialize.go

```diff
@@ -0,0 +1,19 @@
+package tags
+
+import (
+	"fmt"
+	"strings"
+
+	mesh_proto "github.com/Kong/kuma/api/mesh/v1alpha1"
+)
+
+func Serialize(tags mesh_proto.MultiValueTagSet) string {
+	var pairs []string
+	for _, key := range tags.Keys() {
+		pairs = append(pairs, fmt.Sprintf("&%s=%s&", key, strings.Join(tags.Values(key), ",")))
+	}
+	if len(pairs) == 0 {
```

Comment:
is this if necessary? 
https://play.golang.org/p/CTr0tpuPYDs

Reply:
Sure, there were `return "&" + strings.Join(pairs, "&") + "&"` and I forgot to delete that `if`